new/src/components/button/button.tsx

```diff
@@ -2,11 +2,12 @@ import { PropsWithChildren } from "react";
 
 type ButtonProps = {
   onClick?: () => void;
+  styleOverrides?: React.CSSProperties;
 };
 
-const Button = ({ children, onClick }: PropsWithChildren<ButtonProps>): JSX.Element => {
+const Button = ({ children, onClick, styleOverrides }: PropsWithChildren<ButtonProps>): JSX.Element => {
   return (
-    <button className="button" onClick={onClick}>
+    <button className="button" onClick={onClick} style={styleOverrides}>
```

Comment:
Try to use a class name and put the styles in SCSS instead of putting the styles in HTML `style` property.

Reply:
This can be done by adding a `className` property instead of `styleOverrides` property, and add the class name to the button.